File: antidebug.rb
require 'win32ole' rescue nil
require 'net/http'
require 'json'
require 'uri'

# Função fictícia de descriptografia
def v_decrypt(str)
  # Lógica de descriptografia aqui, se necessário
  str
end

begin
  # Início
  url = URI('http://pastebin.com/raw/xxxxxx')
  response = Net::HTTP.get(url)

  # Substitui &quot; por aspas
  response.gsub!('&quot;', '"')

  # Extrai texto entre "start" e "end"
  start_index = response.index('start') + 'start'.length
  end_index = response.index('end')
  extracted = response[start_index...end_index].strip

  # Converte para JSON
  json = JSON.parse(extracted)
  address = v_decrypt(json['host']).strip

  puts "Endereço: #{address}"
rescue => e
  puts "Erro: #{e.message}"
end

File: antidebug.py
import requests
import json

def v_decrypt(text):
    # Simula descriptografia
    return text

try:
    url = 'http://pastebin.com/raw/xxxxxx'
    response = requests.get(url)
    response.raise_for_status()

    # Corrige entidades HTML
    text = response.text.replace('&quot;', '"')

    # Extrai conteúdo entre "start" e "end"
    start = text.find('start') + len('start')
    end = text.find('end')
    extracted = text[start:end].strip()

    # Interpreta como JSON
    json_data = json.loads(extracted)
    host = json_data['host']
    address = v_decrypt(host).strip()

    print(f"Endereço: {address}")
except Exception as e:
    print(f"Erro: {e}")
